commit fe62578c0a6803960aa2f72b9c2693dbcb721a95
Author: Moss <ramivalmont@gmail.com>
Date:   Sat Jul 24 22:14:33 2021 +1200

    MDL-72170: Fix tick box location on quizes
    
    Adjusted position so that tick boxes are next to answers for multichoice and T/F questions

diff --git a/question/type/multichoice/renderer.php b/question/type/multichoice/renderer.php
index 5ceba14536a..689471e4b29 100644
--- a/question/type/multichoice/renderer.php
+++ b/question/type/multichoice/renderer.php
@@ -110,10 +110,10 @@ abstract class qtype_multichoice_renderer_base extends qtype_with_combined_feedb
                         $this->number_in_style($value, $question->answernumbering), 'answernumber');
             }
             $answertext = $question->format_text($ans->answer, $ans->answerformat, $qa, 'question', 'answer', $ansid);
-            $questionanswer = html_writer::div($answertext, 'flex-fill ml-1');
+            $questionanswer = html_writer::div($answertext, 'flex-fill ml-1 mr-1');
 
             $radiobuttons[] = $hidden . html_writer::empty_tag('input', $inputattributes) .
-                    html_writer::div($questionnumber . $questionanswer, 'd-flex w-100', [
+                    html_writer::div($questionnumber . $questionanswer, 'd-flex', [
                         'id' => $inputattributes['id'] . '_label',
                         'data-region' => 'answer-label',
                     ]);
